lib/l4p/tle_udp.h

```diff
@@ -98,6 +98,19 @@ int
 tle_udp_stream_get_param(const struct tle_stream *s,
 	struct tle_udp_stream_param *prm);
 
+/**
+ * Get current UDP stream RX queue size
+ * @param us
+ *   Stream to retrieve state information from.
+ * @return
+ *   On success, the value will be >= 0 and can safely be stored
+ *   in an uint32_t. This value will be the RX queue size of the
```

Comment:
You're returning an int64_t.  Why are you telling the user they can store an unsigned integer in a container that COULD conceivably be out side of the range?

Also, I think it should read "in a uint32_t" versus "in an uint32_t"

Reply:
On success it can be safely stored in a uint32_t.  error codes are negative values.  i can't return an error code and support  the full range of the field using a int32_t so the size is bumped to a int64_t to allow negative error codes and the full range of the field on success.

Reply:
Understood, I'd still fix the English in there to use `a` versus `an` properly.

Reply:
feel like I'm missing something on this.  `an unsigned integer..` vs `a unsigned integer...`?

Reply:
In general "a" goes before words that begin with consonants. "An" goes before words that begin with vowels: an apricot. an egg.

https://owl.purdue.edu/owl/general_writing/grammar/articles_a_versus_an.html